src/tests/baseservices/ilasm_ildasm/regression/vswhidbey305155/305155.cs

```diff
@@ -1,15 +1,17 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 using System;
+using Xunit;
 
 [AttributeUsage(AttributeTargets.Method)]
 public class MyAttribute : Attribute
 {
-	public Type[] Types;
+    public Type[] Types;
 }
 
 public class Test
 {
-	[MyAttribute(Types = new Type[]{typeof(string), typeof(void)})]
-	public static int Main() { return 0; }
+    [MyAttribute(Types = new Type[]{typeof(string), typeof(void)})]
+    [Fact]
+    public static void TestEntryPoint() { }
```

Comment:
This test is marked as a library (was it running before?), which I think will now break under BuildAsStandalone and can just be removed from the project.  I don't know if other Library annotations under baseservices are accurate.

Reply:
I'll try to double-check, I think that for the merged tests we default to Library OutputMode.

Reply:
Oh, I think you're right.  We override it (questionably, but beyond the scope there of course) to Exe when BuildAsStandalone is set.  So it's ok but just redundant.

Reply:
I have removed the superfluous Library OutputType annotation, thanks for spotting.